internal/providers/terraform/terragrunt_hcl_provider.go

```diff
@@ -678,15 +677,7 @@ func (p *TerragruntHCLProvider) runTerragrunt(opts *tgoptions.TerragruntOptions)
 	}
 
 	mod := h.Module()
-	if mod.Error != nil {
-		path := ""
-		if mod.Module != nil {
-			path = mod.Module.RootPath
-		}
-		info.addWarning(schema.NewDiagTerragruntModuleEvaluationFailure(mod.Error))
-		p.logger.Warn().Msgf("Terragrunt config path %s returned module %s with error: %s", opts.TerragruntConfigPath, path, mod.Error)
-	}
```

Comment:
I'm not sure exactly why we treated these errors as warnings previously, I can't find anything. However my assumption is that this logic worked previously in Infracost cloud but we at some point changed the behaviour to be less stringent with "warnings". 

I think treating these errors as proper "fail" errors is a safe and correct change, I don't believe these should be warnings.

Reply:
Agree, not sure why this was the case. These should be errors.